osx/roles/osx.rb

```diff
@@ -0,0 +1,2 @@
+include_recipe '../cookbooks/vim/default.rb'
+include_recipe '../cookbooks/android/default.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.